SchedulerJobs/SchedulerJobs.Common/Constants/AzureAdUserRoles.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SchedulerJobs.Common.Constants
```

Comment:
Should this be an enum rather than constants?

Reply:
Unfortunately C#, unlike Typescript, does not support string enums, only integers, which is why this is a constant.